src/app/[locale]/specs/lexicon/en.mdx

```diff
@@ -208,6 +231,34 @@ Note that unlike `object`, there is no `nullable` field on `params`.
 
 `params` can not be declared as a top-level named type in a schema `defs` array. This means that they can not be referenced by a `ref` or `union`.
 
+### `permission`
+
+This is a limited-scope type which is only ever used for the `permissions` field of the `permission-set` primary type, and should not appear in other schema contexts. They map to auth permissions.
+
+Type-specific fields:
+
+- `resource` (string, required): indicates what account resource is being accessed
+
+Additional `permission` fields depend on the resource type being declared. The set of resources may expand over time, but currently includes:
+
+- `repo`: public repository write permissions
+    - `collection` (array of strings, required): indicates record types. Must include at least one element (not empty array). Supports wildcard (`*`), but does not support partial-NSID glob patterns (`com.example.*`)
```

Comment:
```suggestion
    - `collection` (array of unique strings, required): indicates record types. Must include at least one element (not empty array). Wildcard (`*`) or partial-NSID glob patterns (`com.example.*`) are **NOT** allowed.
```

Wildcards are actually **NOT** allowed here. The reason is that permission-sets can only reference NSIDs **under** (as in, same domain or strict subdomain) the same authority as the permission-set itself.

Any permission item containing `*` here **must** be rejected (TS implementation will ignore those)

We _could_, in the future, relax this to allow `*` here to mean `<nsid-group>.*` but this makes implementation harder since we don't allow partial wildcards in regular permissions.